### PR TITLE
Add support for disabling Gitub wiki, issues and projects

### PR DESCRIPTION
**Pull request by sfc-gh-kleonhard**
## Hey, I just made a Pull Request!

This PR adds three new Github repo options: hasWiki, hasIssues, hasProjects.

The default value for these options remains as is, unset, to avoid any behavior change. API reference: https://octokit.github.io/rest.js/v19

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/sharp-poets-count.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```
Comment (jhaals) [inline .changeset/sharp-poets-count.md:2]:
```suggestion
'@backstage/plugin-scaffolder-backend': minor
```

Comment (jhaals) [reply .changeset/sharp-poets-count.md:2]: Let's make sure this is a minor bump as the package is version 1.x

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks!

File: .changeset/sharp-poets-count.md (hunk 1 above)
Comment (sfc-gh-kleonhard) [reply .changeset/sharp-poets-count.md]: Done.

**Review by jhaals — APPROVED** [review]
Thanks!

**Pull request merged**